copy_encrypted_ami.sh

```diff
@@ -27,6 +27,7 @@ usage()
     -n,               Enable ENA support on new AMI. (Optional)
     -t,               Copy Tags. (Optional)
     -k,               Specific KMS Key ID for snapshot re-encryption in target AWS account. (Optional)
+    -u                Set this value to tag Env for the destination image (Optional). Valid only with -t
```

Comment:
I like the idea, however I think we should be able to add any Tag name rather than specifically "Env" as this might not be the standard for other users. Can you take the Tag Name as well from the input?

Reply:
Added in new version. 
I take a value like "tagname:tagvalue"
It works for existing (then update value) and non existing (then add tag)
I just added a check to ensure the separator ":" is present to avoid aws issue will null values.
No other check (like non permitted characters if that makes sense).